### DISCUSSION TITLE
Plans for downloadable reports

### DISCUSSION THREAD

**Discussion by JayAtFujifilm** [DependencyTrack/dependency-track#1439] [Ideas]
Are there any plans to add concise, downloadable reports (perhaps in PDF format) to DependencyTrack?

**Comment by stevespringett** [comment]
There are no plans to add a reporting engine. There are many open source and commercial reporting engines available, along with general purpose BI tools. Many organizations leverage existing reporting and BI tools and simply integrate them with the Dependency-Track API.

**Comment by JayAtFujifilm** [comment]
Thank you, Steve. That is helpful for us to know how to proceed with reporting.
